Directory.Build.props

```diff
@@ -36,14 +36,15 @@
 
     <!-- Version information -->
     <ScalarVersion>0.2.173.2</ScalarVersion>
+    <MinimumGitVersion>v2.25.0.vfs.1.1</MinimumGitVersion>
```

Comment:
I see we are adding this even though it is present further down. Something to clean up when we actually merge this.

Reply:
Hmm. Let me try to remove it.